data/editor.config

```diff
@@ -444,6 +444,32 @@
         <include group="poi" />
         <include field="internet" />
       </type>
+      <type id="healthcare-alternative" group="health">
+        <include group="poi" />
+        <include field="internet" />
+      </type>
+      <type id="healthcare-audiologist" group="health">
+        <include group="poi" />
+      </type>
+      <type id="healthcare-blood_donation" group="health">
+        <include group="poi" />
+        <include field="internet" />
+      </type>
+      <type id="healthcare-optometrist" group="health">
+        <include group="poi" />
```

Comment:
We discussed that a new group "poi_internet" should be added and used for all locations where it is possible to have a public or "for visitors only" internet access.

Reply:
So should I add this group to all new healthcare POI's ?
Or should I remove the internet field and the new group will be added later?

Reply:
You can change it to poi_internet now, we'll merge this PR when #4050 will be merged.

Reply:
Okay added in https://github.com/organicmaps/organicmaps/pull/4053/commits/d647b9e7369265917c07c1a2dbc4df9b1871ce5c